src/riak_cs_console.erl

```diff
@@ -100,10 +102,84 @@ cleanup_orphan_multipart(Timestamp) when is_binary(Timestamp) ->
     _ = io:format("~nAll old unaborted orphan multipart uploads have been deleted.~n", []).
 
 
+%% @doc Resolve siblings as quick operation. Assuming login via
+%% `riak-cs attach` RawKey and RawBucket should be provided via Riak
+%% log. Example usage is:
+%%
+%% > {ok, Pid} = riakc_pb_socket:start_link(Host, Port).
+%% > riak_cs_console:resolve_siblings(Pid, <<...>>, <<...>>).
+%% > riakc_pb_socket:stop(Pid).
+%%
+%% Note: this does not support multibag itself
+-spec resolve_siblings(pid(), RawBucket::binary(), RawKey::binary()) -> ok | {error, term()}.
+resolve_siblings(Pid, RawBucket, RawKey) ->
+    GetOptions = [{r, all}],
+    _ = lager:info("Trying to resolve siblings of ~p:~p", [RawBucket, RawKey]),
+    case riakc_pb_socket:get(Pid, RawBucket, RawKey, GetOptions, ?DEFAULT_RIAK_TIMEOUT) of
+        {ok, RiakObj} ->
+            case resolve_ro_siblings(RiakObj, RawBucket, RawKey) of
```

Comment:
Output value count is printed out especially in case of manifests. That should depend on object types and not outputtting right here.